string joint_name

HysteresisRun[] runs

string[] arg_name
float32[] arg_value
